packages/mobile/src/account/FiatExchange.tsx

```diff
@@ -80,16 +76,27 @@ class FiatExchange extends React.Component<Props, State> {
   }
 
   render() {
-    return this.state.signedUrl === '' ? (
-      <ActivityIndicator size="large" color={colors.celoGreen} />
-    ) : (
-      <WebView style={styles.exchangeWebView} source={{ uri: this.state.signedUrl }} />
+    return (
+      <SafeAreaView style={styles.background}>
+        <DrawerTopBar />
```

Comment:
I notice you are removing navigationOptions often and placing the topbar directly inside of the component. 

why?

Reply:
The drawer navigator unfortunately doesn't have a header bar, so we could either put a new stack inside each drawer, or just do this - which I think is more performant and easier